src/webpack/executeWebpack.ts

```diff
@@ -17,31 +18,44 @@ import {getRootDirectory} from '../utils/getRootDirectory';
 export async function executeWebpack(mode: WebpackMode, writeToDisk: boolean): Promise<RequestHandler> {
     return new Promise<RequestHandler>((resolve) => {
         // Load config and set development mode
-        const config: webpack.Configuration[] = require(`${getRootDirectory()}/webpack.config.js`);
+        const config: Configuration|Configuration[] = require(`${getRootDirectory()}/webpack.config.js`);
 
-        // eslint-disable-next-line
-        config.forEach((entry: webpack.Configuration) => entry.mode = (entry.mode || mode));
+        // Ensure 'mode' is set on all entry points
+        if (Array.isArray(config)) {
+            config.forEach((entry: Configuration) => {
+                entry.mode = (entry.mode || mode);
+            });
+        } else {
+            config.mode = (config.mode || mode);
+        }
 
         // Create express middleware
-        const compiler = webpack(config);
+        const compiler = webpack(config as Configuration);
```

Comment:
This is to do with `webpack` not being typed properly? Would add a comment if so

Reply:
The types allow either `Configuration` or `Confguration[]`, but get confused by us using `Configuation|Configuration[]`.

Doesn't matter either way though, you get a compiler object back regardless. Also added a type to `compiler`, to make sure we're taking the common denominator between the different possible return types of `webpack`.